src/include/block_chunkcache.h

```diff
@@ -0,0 +1,74 @@
+/*-
+ * Copyright (c) 2014-2020 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *  All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WiredTiger's chunk cache. Locally caches chunks of remote objects.
+ */
+
+#define WT_CHUNKCACHE_DEFAULT_HASHSIZE 32 * 1024
+#define WT_CHUNKCACHE_DEFAULT_CHUNKSIZE 1024 * 1024
+#define WT_CHUNKCACHE_FILE 1
+#define WT_CHUNKCACHE_IN_VOLATILE_MEMORY 0
+#define WT_CHUNKCACHE_MINHASHSIZE 64
+#define WT_CHUNKCACHE_MAXHASHSIZE 1024 * 1024
+#define WT_CHUNKCACHE_MAX_RETRIES 1024 * 1024 * 1024
+#define WT_CHUNKCACHE_NAMEMAX 50
+#define WT_CHUNKCACHE_UNCONFIGURED 0
+
+struct __wt_chunkcache_hashid {
+    char objectname[WT_CHUNKCACHE_NAMEMAX];
+    uint32_t objectid;
+    wt_off_t offset;
+};
+
+/*
+ * The encapsulation of a cached chunk.
+ */
+struct __wt_chunkcache_chunk {
+    TAILQ_ENTRY(__wt_chunkcache_chunk) next_chunk;
+    TAILQ_ENTRY(__wt_chunkcache_chunk) next_lru_item;
+
+    WT_CHUNKCACHE_HASHID hash_id;
+    volatile bool being_evicted;
+    char *chunk_memory;
+    size_t chunk_size;
+    unsigned int bucket_id; /* save bucket ID for quick removal */
```

Comment:
Similarly we're using `bucket` for tiered storage. Is there a different name you could use? Maybe call it `hash_id` and comment `hash bucket ID for quick removal`??

Reply:
Updated the comment. Kept the name for readability.